src/network/network.cpp

```diff
@@ -575,6 +576,11 @@ void NetworkClose(bool close_admins)
 /* Initializes the network (cleans sockets and stuff) */
 static void NetworkInitialize(bool close_admins = true)
 {
+	/* Make sure we always have a name. */
```

Comment:
This change can now go, otherwise it breaks the do not use "Player" as default anymore.